### ISSUE TITLE
[Feature Request] User and Administrator Documentation

### ISSUE THREAD

**Issue by wagnered** [ampache/ampache#2315]
The wiki is mostly for the administrator during installation and it does a fair job.  But for actual administration, there is almost no explanation for the items on the admin pages he/she has access to. 

Speaking for myself, as a developer, I think I have managed to learn about most of the features, but as a user, I don't think I know every feature that Ampache provides or how to use them.

This document  would be organized for the user and administrator and would explain every item or feature in every view in an organized manner.  It would make no assumptions about the user or administrator's knowledge of Ampache.

The documentation for the user could be limited to  the home page and user preferences page.  

All complex applications have some level of help, built in or on line. I think that Ampache qualifies in complexity enough to warrant user documentation.  A user should not have to "go through channels" to get the listening pleasure he or she deserves.

**Comment by kuzi-moto** [comment]
I completely agree, and had previously been thinking of adding in some basic user guides for navigating the interface as a normal user.

The only reason I have waited was because I think Ampache needs a face-lift and didn't want to spend a bunch of time making screenshots and stuff for the reborn interface, only to have to re-do it later. But I'm not sure when that will happen so maybe it's better to do it anyway.

**Comment by lachlan-00** [comment]
agreed on all. the options / interface wont change that drastically in the near future.

even if it did the concepts shouldn't be changing beyond a visual.
